monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md

```diff
@@ -1,12 +1,12 @@
 # MONERO
 
-Monero es una moneda digital segura, privada y no rastreable disponible para todos.
+Monero es una moneda digital segura, privada y no rastreable disponible para todos. Sus características lo hacen dinero fungible.
 
 # HOJA DE DATOS
 
-Los usuarios pueden gastar de una manera segura, nadie puede ver sus balances ni rastrear sus actividades.
+Los usuarios pueden gastar de una manera segura, nadie puede ver sus balances ni rastrear sus actividades a menos que el usuario les de el permiso.
 
-Revisado 11/14/2018 - [Versión para imprimir](http://www.monerooutreach.org/pubs/2018/QuickFacts/Quick-facts_es_version-para-imprimir.pdf)  
+Revisado 2/1/2019 - [Versión para imprimir](http://www.monerooutreach.org/pubs/2018/QuickFacts/Quick-facts_es_version-para-imprimir.pdf)  
```

Comment:
The date is not consistent with the other dates. 01/Feb/2019.

Reply:
Got it :+1: